### ISSUE TITLE
Kokkos Reax Package Development Plan

### ISSUE THREAD

**Issue by jimkress65** [lammps/lammps#551]
Is there any working being done on (or planned for) the Kokko Reax package to incorporate ACKS2 and eReaxFF?

**Comment by akohlmey** [comment]
Please post questions about LAMMPS to the lammps-users mailing list. We would like to keep this forum reserved for bug reports and discussions between developers and contributors. Thanks for your understanding.

**Comment by stanmoore1** [comment]
Or it is also fair to rephrase this as an enhancement request and post it here on GitHub, like "Incorporate ACKS2 and eReaxFF into Kokkos Reax package".

**Comment by jimkress65** [comment]
> Or it is also fair to rephrase this as an enhancement request and post it here on GitHub, like "Incorporate ACKS2 and eReaxFF into Kokkos Reax package".

That would be fine with me.  I figured I’d get yelled at for posting it in the LAMMPS forum, since it is KOKKOS specific.

**Comment by akohlmey** [comment]
It is a bit of a borderline case with no straightforward response. 

i don't mind seeing a well formulated feature request here with a suitable subject line and a good motivation. but unless it is a feature, that is obviously useful and easy to implement, or you are offering to participate in implementing it, chances are small to receive the kind of response you are looking for. at least, that is what it looks like when i am looking at the issues that were filed so far. also, please keep in mind, that not all LAMMPS developers currently pay as much attention to the github issue tracker as they do to the lammps-users mailing list. i don't think being kokkos-specific makes it off-topic for the mailing list. after all, you are not asking to have your research explained to you or similar. the most useful posts here so far have been either bug reports or discussions among developers/contributors.

feature requests are labeled "enhancement", so you can see some examples at  https://github.com/lammps/lammps/issues?utf8=%E2%9C%93&q=is%3Aissue%20label%3Aenhancement%20

what tipped the scales toward closing the issue is, that i am very worried about setting a precedent that may encourage others to ask **definitely** off-topic questions on the issue tracker (like "how do i do XXX?" or "what does this error mean?" and so on).

i hope that explains it a bit better, why i think posting to the mailing list would be the more suitable choice in this particular case.

**Comment by jimkress65** [comment]
Axel,

 

Thank you for your detailed response.  I submitted the question to the LAMMPS list as you suggested.

 

Jim
